README.md

```diff
@@ -1,9 +1,190 @@
 # clever_tools
 
-Fun ROS nodes and stuff for CLEVER drone kit, that is not included to the [Raspberry Pi image](https://github.com/CopterExpress/clever/releases).
+Useful ROS nodes and stuff for CLEVER drone kit, that is not included to the [Raspberry Pi image](https://github.com/CopterExpress/clever/releases).
+
+## Installation
+
+Clone repository to the catkin workspace sources:
+
+```bash
+cd ~/catkin_ws/src
+git clone https://github.com/CopterExpress/clever_tools.git
+```
+
+Execute catkin_make in catkin workspace directory:
+
+```bash
+cd ~/catkin_ws
+catkin_make clever_tools
+```
 
 ## Nodes
 
 ### interactive.py
 
 Control the drone using [interactive markers](http://wiki.ros.org/interactive_markers) in Rviz!
+
+```bash
+rosrun clever_tools interactive.py
+```
+
+### create_route.py
+
+Create flying route by moving drone in space and recording its coordinates by triggering RC pitch stick (channel 2).
+
+```bash
+usage: rosrun clever_tools create_route.py [-h] [-f FRAME_ID] [filename]
+
+positional arguments:
+  filename              Filename of route csv table. Default is route.csv.
+
+optional arguments:
+  -h, --help            show help message and exit
+  -f FRAME_ID, --frame_id FRAME_ID
+                        Coordinates will be recorded relative to frame_id
+                        parameter. Default is map.
+```
+
+### fly_route.py
+
+Fly route from the csv table with x y z values.
+
+```bash
+usage: rosrun clever_tools fly_route.py [-h] [-r] [-z HEIGHT] [-f FRAME_ID] [-s SPEED] [filename]
+
+positional arguments:
+  filename              Filename of the route csv table
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -r, --repeat          Program will repeat the route infinite if you use this
+                        arg.
+  -z HEIGHT, --height HEIGHT
+                        Height of flight. Default is 1 m. If height is 'nan'
+                        then the drone will takeoff to 1 m and fly on the
+                        route point heights.
+  -f FRAME_ID, --frame_id FRAME_ID
+                        Coordinates will be used relative to frame_id
+                        parameter. Default is map.
+  -s SPEED, --speed SPEED
+                        Copter speed. Default is 1 m/s.
+```
+
+### test_flight.py
+
+Just example node of flight: takeoff to 1m, fly forward 1m, and land.
+
+## Python modules
+
+### flight_routines
+
+#### Usage example
+
+```
+import rospy
+from flight_routines import *
```

Comment:
Actually, importing star considered bad practice.

Reply:
Changed this in https://github.com/CopterExpress/clever_tools/pull/2/commits/9a171f9f7992e75b708a2f44de95b9eaddfa2d2d